.github/workflows/pr.yml

```diff
@@ -40,3 +40,26 @@ jobs:
       - name: Gulp (iOS)
         run: npx gulp buildIOS
         working-directory: ./Package
+
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2.3.3
+        with:
+          submodules: 'true'
```

Comment:
Is this `true` instead of `recursive` specifically for the ARCore dependency? Don't we have other nested submodules, like bgfx under bgfx.cmake? How does this work?

Reply:
this should be handled by the powershell script as you noted later in the review